addons/web/static/src/core/commands/command_palette.js

```diff
@@ -62,11 +62,13 @@ function commandsWithinCategory(categoryName, categories) {
 
 export class DefaultCommandItem extends Component {}
 DefaultCommandItem.template = "web.DefaultCommandItem";
+let lastSessionId = 0;
 
 export class CommandPalette extends Component {
     setup() {
         this.keyId = 1;
         this.keepLast = new KeepLast();
+        this._sessionId = lastSessionId++;
```

Comment:
Could you add a test ?
Open the command palette check id pass to provider.
Then close it.
Open the palette command check that it is id + 1.

Reply:
Done.